src/SpaceExploration.Game/Scoring/Scoring_1_0_10_0.cs

```diff
@@ -0,0 +1,93 @@
+
+
+using SpaceExploration.Game.Contracts.Drones.Events;
+using SpaceExploration.Game.Contracts.Planets.Commands;
+
+namespace SpaceExploration.Game.Scoring;
+
+public class Scoring_1_0_10_0 : Saga<SocringData>,
+    IAmStartedByMessages<Create_1_0_10_0_Scoring>,
+    IHandleMessages<DroneDropped>,
+    IHandleMessages<DroneHit>,
+    IHandleMessages<DroneDestroyed>
+{
+
+    public Task Handle(Create_1_0_10_0_Scoring message, IMessageHandlerContext context)
```

Comment:
Is it better to start scoring by an explicit command or by handling the PlanetCreated event?

By handling the event for each planet every possible scoring would be started. This would yield to duplicate DroneScoreUpdated events. 

Doing it by an explicit command gives more flexibility.

Reply:
For now stick with command